sdk/cosmos/azure-cosmos/azure/cosmos/aio/_container.py

```diff
@@ -282,7 +282,8 @@ async def read_item(
         :param item: The ID (name) or dict representing item to retrieve.
         :type item: Union[str, Dict[str, Any]]
         :param partition_key: Partition key for the item to retrieve.
-        :type partition_key: Union[str, int, float, bool, List[Union[str, int, float, bool]]]
+        :type partition_key: Union[str, int, float, bool, List[Union[str, int, float, bool, None]],
+            Tuple[Union[str, int, float, bool, None]]]
```

Comment:
Does this now allows customers to pass Tuple?
Or it always allowed, we just threw error in that case?

Reply:
Customers could always pass in any value they wanted since python doesn't enforce typing. In the type hints however, it was labeled with Sequences, but sequences technically imply we allow tuples and even further than that it would imply things such as Strings, Bytes, Ranges, etc. So I am making it more explicit and saying the user should use a list or a tuple.